PowerVCF.psm1

```diff
@@ -104,7 +104,7 @@ public static class Dummy {
             $response = Invoke-RestMethod -Method POST -Uri $uri -Headers $headers -body $body -SkipCertificateCheck # PS Core has -SkipCertificateCheck implemented
             $Global:accessToken = $response.accessToken
             $Global:refreshToken = $response.refreshToken.id
-        }
+        }:
```

Comment:
What's the : for?

Reply:
sorry that's clearly a typo, my bad. easy fix :-)

Reply:
fixed